ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -56,6 +56,16 @@ VAV_SD_HLSAFS:
   - SD
   - HLSAFS
 
+VAV_SD_HLSAFS_CO2C:
```

Comment:
Please confirm that there is no temperature control.

Reply:
Hi Shamber - no temp control, control based on the co2 setpoint